app/templates/explore.hbs

```diff
@@ -1,24 +1,42 @@
+<style>
+    .collection-card-content {
+        flex: 1 0 auto;
+    }
+    .card.coll-single {
+        display: flex;
+        flex-direction: column;
+    }
+</style>
 {{outlet}}
 {{nav-bar}}
 <div class="coll-banner text-center">
-        <h1 class="f-w-lg"> OSF Collections </h1>
-        <p class="lead f-w-md">Organize your research interests in a way that makes sense</p>
-        {{#if session.isAuthenticated}}
-            {{#link-to 'create' class="btn btn-success btn-lg m-t-lg"}}Start a collection {{/link-to}}
-        {{/if}}
+    <h1 class="f-w-lg"> OSF Collections </h1>
+    <p class="lead f-w-md">Organize your research interests in a way that makes sense</p>
+    {{#if session.isAuthenticated}}
+        {{#link-to 'create' class="btn btn-success btn-lg m-t-lg"}}Start a collection {{/link-to}}
+    {{/if}}
 </div>
 
 <h2 class="text-center">Newest collections</h2>
 <div class="cards">
     {{#each model as |collection|}}
         <div class="card coll-single">
-            {{#link-to "collections.collection" collection.id class="btn btn-info pull-right"}}<span {{action 'clearFilter'}}>View</span>{{/link-to}}
-            <h3>{{#link-to "collections.collection" collection.id}}<span {{action 'clearFilter'}}>{{collection.title}}</span>{{/link-to}}</h3>
-            <p>{{truncate-text collection.description limit=250 omission="..."}}</p>
-            {{#if collection.tags}}
-                <div>Tags: {{collection.tags}}</div>
-            {{/if}}
-            <div class="text-muted">Created on {{moment-format collection.dateCreated "MMMM DD, YYYY"}} </div>
+            <div class="collection-card-content">
+                {{#link-to "collections.collection" collection.id class="btn btn-info pull-right"}}
+                    <span {{action 'clearFilter'}}>View</span>{{/link-to}}
+                <h3>{{#link-to "collections.collection" collection.id}}
+                    <span {{action 'clearFilter'}}>{{collection.title}}</span>{{/link-to}}</h3>
+                <p>{{truncate-text collection.description limit=250 omission="..."}}</p>
```

Comment:
We could just turn this p into a div and put `flex: 1 0 auto;` on that, `display: flex` on `.card.coll-single`. It would remove the need for `collection-card-content` and `collection-card-footer`.

Reply:
Actually probably dont even need to change the element name, just put that style on the p